app/assets/stylesheets/frontend/layouts/html-publication.scss

```diff
@@ -26,7 +26,7 @@
   }
 }
 
-#meta-data {
+#print-meta-data {
```

Comment:
This is pedantic, but these changes (where you rename css id's, css classes, and file names from `meta-data` to `print-meta-data`) would be better squashed down into the previous commit.  They're really not much to do with this commit because if for some strange reason we decided we didn't want the new fields you add here we'd still want the classes and such to be called `print-meta-data`.